Dockerfile.WorkerSetupJob

```diff
@@ -0,0 +1,19 @@
+FROM registry.access.redhat.com/ubi8/python-39
+
+USER 0
+
+COPY resources/ansible/requirements.yml /tmp/requirements.yml
+
+RUN dnf install skopeo -y && \
+    pip install --upgrade pip && \
+    pip install openshift && \
+    pip install jmespath && \
+    pip install netaddr && \
+    pip install ansible==2.9.* && \
```

Comment:
I think it's better to use a specific version that's working for you in the lab environment

Reply:
Done